src/main/java/sirius/biz/storage/layer3/Transfer.java

```diff
@@ -331,26 +331,29 @@ private void transfer(boolean delete) {
                 destination.createAsDirectory();
             }
 
-            transferDirectory(source, destination.findChild(source.name()), delete);
+            transferDirectory(source, destination.findChild(source.name()), delete, delete);
```

Comment:
Even if thats correct this looks like an error, so maybe explain why the same param is passed twice here :)